backends/magma/ceed-magma-basis.c

```diff
@@ -414,7 +431,7 @@ static int CeedBasisApplyNonTensor_Magma(CeedBasis basis, CeedInt num_elem, Ceed
     } else {
       for (CeedInt d = 0; d < q_comp; d++) {
         if (t_mode == CEED_TRANSPOSE) {
-          const CeedScalar beta = (d > 0) ? 1.0 : 0.0;
+          const CeedScalar beta = apply_add || ((d > 0) ? 1.0 : 0.0);
```

Comment:
Wouldn't this be cleaner if we did `(apply_add || (d > 0)) ? 1.0 : 0.0` ? To avoid having a scalar value on one side of the `||`?